onchain/src/art_peace.cairo

```diff
@@ -88,6 +98,7 @@ pub mod ArtPeace {
         };
 
         self.creation_time.write(starknet::get_block_timestamp());
+        self.start_day_time.write(starknet::get_block_timestamp());
```

Comment:
@b-j-roberts  do you prefer the redondant use of get_block_timestamp or is it better to assign to self.creation_time.read() ?

Reply:
No, this is good! It seems clearer like this, too.